status-im/status-react
# Chat views redesign

fixes #13156 

### Summary

Chat views redesign

<img width="448" src="https://user-images.githubusercontent.com/18485527/175847465-783b5e99-b23f-48c4-b32e-4807b1147939.png">

### Review notes

Read testing notes

### Testing notes

Redesign is still a WIP and there are lots of changes to work on so there still will be inconsistencies between designs and new design implementation. All the changes are under the New UI toggle. The idea is to merge this as soon as possible so we can work on each messages components, other UI changes and fixes in isolation, as this PR became bigger than expected. As long as this PR doesn't break the existing UI I would merge and continue working on redesigns.

#### Platforms

- Android
- iOS

#### Areas that maybe impacted

Chats, messages

##### Functional

- 1-1 chats
- public chats
- group chats

### Steps to test

- Open app
- Switch to New UI
- Check chats list and messages

status: ready
